lifelines/fitters/coxph_fitter.py

```diff
@@ -299,6 +299,7 @@ def fit(self, df, duration_col, event_col=None,
         self._check_values(df, E)
         df = df.astype(float)
         self.data = df if self.strata is None else df.reset_index()
+        self._n_examples = self.data.shape[0]
```

Comment:
if we are to do this, I would suggest not giving CoxPHFitter the `data` attribute at all. So this would be 
```
self._n_examples = df.shape[0]

```